src/main/kotlin/dev/stalla/model/googleplay/ExplicitType.kt

```diff
@@ -0,0 +1,32 @@
+package dev.stalla.model.googleplay
+
+/**
+ * Enum model for the defined values encountered within the
+ * `<googleplay:explicit>` element within a `<item>` element.
+ *
+ * @property type The string representation of the enum instance.
+ */
+public enum class ExplicitType(public val type: String) {
+
+    /** Type describing a `yes` value. */
+    YES("yes"),
+
+    /** Type describing a `no` value. */
+    NO("no"),
+
+    /** Type describing a `clean` value. */
+    CLEAN("clean");
+
+    public companion object Factory {
+
+        /**
+         * Factory method for the instance of the Enum matching the [type] parameter.
+         *
+         * @param type The string representation of the Enum instance.
+         * @return The Enum instance matching [type], or null if not matching instance exists.
+         */
+        public fun from(type: String?): ExplicitType? = type?.let {
```

Comment:
Will change to new name after this has been merged

Reply:
My suggestion was to rename `from()` of enums to `of()`, and we decided to rename the `from()` of builders to `applyFrom()`

Reply:
Ah, I thought we were only doing the `applyFrom` change. Sure, not overly attached to the naming for the enums, as long as it's consistent 👍